src/Api/Encoder.php

```diff
@@ -101,35 +108,68 @@ public function run(ImageInterface $image): EncodedImageInterface
      */
     public function getFormat(ImageInterface $image): string
     {
-        $fm = (string) $this->getParam('fm');
```

Comment:
This needs to be preserved. If `fm` is specified then that's the format that should be returned irrespective of the actual media type.

Reply:
it's preserved, its just moved there:
https://github.com/thephpleague/glide/pull/410/files#diff-63a111a0d299d1ea5fa389c1909b9577ed17e30e75c26627802b3cb3c8cdd606R89

Reply:
Well directly calling this method won't take into account `fm` would it?  This is a public API.

Reply:
Sorry, I misread the code, `fm`'s value is accounted for.